src/wp-includes/script-loader.php

```diff
@@ -427,9 +428,33 @@ function wp_default_packages_inline_scripts( $scripts ) {
 						'meridiem'      => (object) $wp_locale->meridiem,
 						'relative'      => array(
 							/* translators: %s: Duration. */
-							'future' => __( '%s from now' ),
+							'future' => __( '%s from now', 'default' ),
```

Comment:
@ramonjd Why use `default` text domain?

Reply:
Eeek, a copy paste error from Gutenberg. Thanks for picking up on that. Will replace.